infra/stacks/gateway.ts

```diff
@@ -23,6 +23,8 @@ const heboGateway = new sst.aws.Service("HeboGateway", {
     LOG_LEVEL: isProd ? "info" : "debug",
     NODE_EXTRA_CA_CERTS: "/etc/ssl/certs/rds-bundle.pem",
     PORT: gatewayPort,
+    BEDROCK_REGION: "us-east-1",
+    VERTEX_LOCATION: "us-central1",
```

Comment:
Since we are not deploying to Bedrock or Vertex, these should probably be part of the secrets (as I guess they need to match the place where the ServiceAccount/Arn/Project/... lives), not part of preconfigured variables.